pyramid_swagger/__init__.py

```diff
@@ -4,20 +4,28 @@
 """
 import pyramid
 from .api import register_api_doc_endpoints
+from .api import register_swagger_json_endpoint
 from .ingest import get_swagger_schema
+from .ingest import get_swagger_spec
 
 
 def includeme(config):
+    """
+    :type config: :class:`pyramid.config.Configurator`
+    """
     settings = config.registry.settings
 
     # Add the SwaggerSchema to settings to make it avialable to the validation
     # tween and `register_api_doc_endpoints`
     settings['pyramid_swagger.schema'] = get_swagger_schema(settings)
+    settings['pyramid_swagger.spec'] = get_swagger_spec(settings)
 
     config.add_tween(
-        "pyramid_swagger.tween.validation_tween_factory",
+        #"pyramid_swagger.tween.validation_tween_factory",   # 1.2
```

Comment:
It will serve docs for 1.2 and 2.0 but all requests will be routed through the 2.0 implementation.  The catch being that view callables have to work for both versions without any changes.